### PR TITLE
Visits screen

### PR DESCRIPTION
**Pull request by mustiwebp**
A list for visits has been created.

### CODE REVIEW

File: lib/screens/my_visits_screen/visit_item.dart [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+              text: TextSpan(
+                text: 'Anfangszeit: ${_stay.record.day}.${_stay.record.month}.'
+                    '${_stay.record.year} ',
+                style: const TextStyle(
+                  color: Colors.black,
+                  fontSize: 18,
+                ),
+                children: <TextSpan>[
+                  TextSpan(
+                    text: 'Endzeit: ${_stay.discharge!.day}.'
```
[the remark this answers is not in the document]
Comment (mustiwebp) [reply lib/screens/my_visits_screen/visit_item.dart]: Where exactly?

File: lib/screens/my_visits_screen/visits_list.dart [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  const VisitsList({Key? key}) : super(key: key);
+
+  @override
+  State<VisitsList> createState() => _VisitsListState();
+}
+
+class _VisitsListState extends State<VisitsList> {
+  @override
+  Widget build(BuildContext context) {
+    return BlocBuilder<ObjectsListBloc<BackendStaysApi>, ObjectsListState>(
```
[the remark this answers is not in the document]
Comment (mustiwebp) [reply lib/screens/my_visits_screen/visits_list.dart:37]: BlocBuilder should be appropriate here.

File: lib/screens/my_visits_screen/visit_item.dart (hunk 1 above)
[the remark this answers is not in the document]
Comment (FaysalSolutions) [reply lib/screens/my_visits_screen/visit_item.dart]: Before the 'end time', so '\nEndTime'

File: lib/screens/my_visits_screen/visits_list.dart (hunk 2 above)
[the remark this answers is not in the document]
Comment (FaysalSolutions) [reply lib/screens/my_visits_screen/visits_list.dart:37]: If you think that no one needs to log in from another device at the same time, then that's okay.

**Review by FaysalSolutions — APPROVED** [review]

**Pull request merged**